### PR TITLE
Solve the r query without the r tree

### PR DESCRIPTION
**Pull request by xcomagent95**
Hi everbody,

I implemented the functions solveQueryWithoutRTree and pointInRegion.
Could one of you please verify if they are working correctly.

Since Jana states that the RQuery should return the trajectories 43, 45, 50, 71, 83 but mine returns a much longer list 
you might assume that my code is wrong (which could be the case of course). But since I am quite confident that it is not
I have written to Jana asking if she could verify the correctness about her result statement.

### CODE REVIEW

File: project/functions_template.py [hunk 1]
```diff
@@ -163,10 +163,10 @@ def solveQueryWithoutRTree(r:region,trajectories:list) -> list:
...
-        for p in t.points:
-            if r.pointInRegion(p):
-                res.append(t)
-                break
-    return res
\ No newline at end of file
+    result = []
+    for t in trajectories: #iterate over trajectories
+        for p in t.points: #iterate over points
+            if r.pointInRegion(p): #Ceck if point lies in region
```
Comment (jbalzer12) [inline project/functions_template.py]: Ceck => Check

Comment (xcomagent95) [reply project/functions_template.py]: Fixed

**Comment by jbalzer12** [comment]
For me it looks like the returned list is the full set of test trajectories. When I visualize the set with the existing functions, it looks like they are all lying above each other. Seems like there is a bug in the import functions maybe ...?

**Comment by xcomagent95** [comment]
I'll have a look into it

**Pull request merged**